benchmarks/systems/system.py

```diff
@@ -22,6 +24,7 @@ class OpenSystem(ClosedSystem):
     def __init__(self):
         super().__init__()
         self.jump_ops = None
+        self.jump_ops_batched = None
```

Comment:
Why is this required in the benchmarks module? This should depend on the benchmarked system if ever we want to have a batched jump_ops.

Reply:
In `test/systems` `self.H_batched` is also defined so it's a matter of consistency. In general it's good practice to define all fields in the `__init__` method of a class in Python